doozerlib/runtime.py

```diff
@@ -241,6 +245,27 @@ def get_named_lock(self, absolute_path):
                 self.dir_locks[p] = new_lock
                 return new_lock
 
+    @staticmethod
+    def check_missing_db_env_var():
```

Comment:
That way essentially `--datastore <env>` informs the DB instead of `DB_NAME` ... I don't think we would want both. Can look up from a constant dict:
```
db_name_for_env = {
    "prod": "build",
    "int": "build_int",
}
```
That will ease the transition a bit too. I can go with that.

Reply:
How do I access <env>? Is it available in runtime? Also assuming default databases or based on <env> should I create the database somewhere if is not exist or just assume it will exist? Otherwise the code starts getting exception everywhere in DBLib.

Reply:
pretty sure `--datastore` populates `runtime.datastore`

Something feels wrong about having doozer create the DB if not existing, but I'm not sure I can articulate it.

Reply:
```
db_name_for_env = {
    "prod": "build",
    "int": "build_int",
}
```
No strong preference on the physical name for the database, but I suggested a generic `doozer_*` because I hope that the team will add more operation types over time. Right now there is just `build`, but `rebase`, `alignment_prs`, etc, can all provide valuable metrics.